cc_plugin_ugrid/checker_10.py

```diff
@@ -27,14 +27,15 @@ def check_mesh_topology_variable(self, ds):
         score = 0
         out_of = 1
         messages = []
-        desc = 'cf_role:mesh_topology variable exists'
+        desc = 'at least one cf_role:mesh_topology variable exists'
 
         mt = ds.get_variables_by_attributes(cf_role='mesh_topology')
         if len(mt) == 1:
             score += 1
         elif len(mt) > 1:
-            m = ('One variable with attribute "cf_role:mesh_topology" allowed')
+            m = ('Multiple grids allowed')
```

Comment:
Don't need a message for a passed test. So you can get rid of this elif block and just do 
```
if len(mt) >= 1:
    score +=1
```